frontend/simple/components/CreateGroup/CreateGroupSummary.vue

```diff
@@ -0,0 +1,116 @@
+<template>
+  <div>
+    <h1 id="summaryStep" class="title is-1 has-text-centered"><i18n>Review & Finish</i18n></h1>
+
+    <h2 class="title is-6"><i18n>Group name:</i18n></h2>
+    <p class="subtitle is-2" v-if="group.groupName">{{ group.groupName }}</p>
+    <p class="subtitle is-2 is-danger" v-else><i18n>No group name set</i18n></p>
```

Comment:
This `v-if`/`v-else` condition shouldn't be necessary, right? By this point the validations should guarantee that `group.groupName` is set?

Reply:
well, as I mentioned, all steps having their own URL makes it pretty easy to end up at this step, even with a simple refresh, that's why I kept these. is that something we should worry about?

Reply:
Hmm, technically you are correct, if the user is using the app in a browser, they could visit this page somehow... without setting the groupName... theoretically, perhaps by looking at the browsing history.

I have two suggestions on how to deal with this:

1. Just make sure that vuelidate doesn't allow the form to be submitted (I'm guessing it already does that? Can you verify?)
2. Have a guard of some sort, perhaps in `mounted ()`, i.e. maybe a mixin that checks the variables that need to be set by this step, and if they aren't set revert to the beginning of the StepAssistant.

Reply:
1. yes, that's the current behavior :)

2. here are my two suggestions:

2.1 the full solution would be to check for every step that all the previous steps are valid, and if not, go back to the first invalid step. this seems like a bigger task, but we have everything for this to work - the step-validator relation and the order of steps
2.2 the quicker solution would be to only guard the summary step, and if it's not valid, jump back to the first step of the assistant. this could work if we assumed that the only way to end up here in an invalid state is by skipping all the previous steps. but it's not the case, someone could easily hit refresh at the 2nd step and only miss the first one. from the user's perspective, without a proper error notification solution #175 , I think this is way more confusing and frustrating than seeing on the summary which step is actually missing and having to manually go back to correct it.

so in my opinion, what we have now is better than 2.2, and I'd love to do 2.1, but I assume we have things with higher priority.
thoughts? :)